README.md

```diff
@@ -144,6 +144,27 @@ Any other layers will be rendered visually but have no impact on the game automa
 additional custom behavior by accessing the layers via `LeapGame.map.tiledMap` and integrating your
 own special behavior for tiles or objects.
 
+#### Customizing layer names and classes
+
+Even though the structure explained above should always be followed, the developer can ask Leap
+to use different classes, types, names.
+
+In order to do so, set a new `LeapOptions` to `LeapOptions.defaults` before loading a map:
+
+```dart
+LeapOptions.defaults = const LeapOptions(
```

Comment:
My only qualm with this PR is the API to set the values...

Setting `LeapOptions.defaults` feels awkward, especially because if you are changing the defaults, does that mean the original values are the "default defaults"?

Anyways, maybe a function instead would be better, like `LeapOptions.changeTiledConstants(...)` ? Or LeapOptions could be a value on LeapGame, and you just set LeapGame.options to a new instance? Leaning towards the second option since the game ref is everywhere anyways

Reply:
Also, it might be good to differentiate between Leap Tiled options and other options (like physics etc), but that's probably premature

Reply:
Yeah, I was on the fence about the naming, I tried a different approach now, let me know what you think of it, I have also prepared for the future in case we want to have multiple type of configurations, not only about tiled.